### PR TITLE
Feature/145 edit testdata

### PR DESCRIPTION
**Pull request by tanmaythole**

### CODE REVIEW

File: src/Components/ProjectsComponent/TeststepComponents/AddNewTestdata.jsx [hunk 1]
```diff
@@ -46,21 +59,41 @@ const AddNewTestdata = ({ data, onChange, onSubmit, handleClose }) => {
...
+      let expOutcome = expectedOutcome;
+      expOutcome.find(
+        (ele) => ele.id === selectedExpOutcome?.value
+      ).expectedOutcome = result;
+      onChange("expectedOutcome", expOutcome);
+    }
   };
 
   // bottom info text for name field
   const nameBottomInfoText =
```
Comment (ChinmaySonawane) [inline src/Components/ProjectsComponent/TeststepComponents/AddNewTestdata.jsx:76]: move this in message constants

File: src/Components/ProjectsComponent/TeststepComponents/TestdataAccordionItem.jsx [hunk 2]
```diff
@@ -10,12 +11,20 @@ import {
...
 
   return (
     <Accordion.Item eventKey={eventKey}>
-      <Accordion.Header>{name}</Accordion.Header>
+      <Accordion.Header>
+        <div className="w-100 d-flex justify-content-between">
+          {name}
+          <PencilSquare
+            className="me-3"
+            onClick={() => onEditButtonClick(data)}
```
Comment (ChinmaySonawane) [inline src/Components/ProjectsComponent/TeststepComponents/TestdataAccordionItem.jsx]: we can write arrow function for this and use it here

Comment (tanmaythole) [reply src/Components/ProjectsComponent/TeststepComponents/TestdataAccordionItem.jsx]: 👍🏻

**Review by ChinmaySonawane — APPROVED** [review]

File: src/Components/ProjectsComponent/TeststepComponents/AddNewTestdata.jsx (hunk 1 above)
Comment (tanmaythole) [reply src/Components/ProjectsComponent/TeststepComponents/AddNewTestdata.jsx:76]: this user message was only used one time

**Pull request merged**